frontend/src/scenes/paths/NewPaths.js

```diff
@@ -114,15 +117,12 @@ export function NewPaths({ dashboardItemId = null, filters = null, color = 'whit
 
         let sankey = new Sankey.sankey()
             .nodeId((d) => d.name)
-            .nodeAlign(Sankey.sankeyJustify)
+            .nodeAlign(Sankey.sankeyLeft)
```

Comment:
@liyiy was there any specific reason for justify?

Left is important (to me) because it keeps paths that end early to the left, instead of at the rightmost end. It messes up endpoint sankey quite a bit 😅

Reply:
I think changing it back to left is fine as long as the cards look okay 👍 Think I was playing around with it originally because @EDsCODE and I discussed that it was super confusing for some node displays to be on both sides (that weren't the last one)